model/accounts_mgmt/v1/billing_model_item_type.model

```diff
@@ -0,0 +1,9 @@
+// BillingModelItem that represents a billing model (defined in pkg/api/billing_types.go). Using BillingModelItem to keep backwards compatibility as we already have a BillingModel enum defined.
+struct BillingModelItem {
+    // Id for the BillingModel
```

Comment:
indentation + missing '.' at end of each comment

Reply:
Added